oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/expression/EqualMatch.java

```diff
@@ -39,19 +41,19 @@ public boolean match(double left, double right) {
     }
 
     public boolean match(Integer left, Integer right) {
-        return left == right;
+        return Objects.equals(left, right);
     }
 
     public boolean match(Long left, Long right) {
-        return left == right;
+        return Objects.equals(left, right);
     }
```

Comment:
`Integer`/`Long` is not the same `Object` when `>128` or so, and should be compared via `equals`, tricky, huh